de.flinkebits.AvaEmailArchivar.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: de.flinkebits.AvaEmailArchivar
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: startup_flatpak.sh
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+
+modules:
+  - name: buildavaemailarchivar
+    only-arches:
+      - x86_64
```

Comment:
you need a `flathub.json` to disable the other architecture.

(or better, build for it)